docs/sources/shared/datasources/tempo-editor-traceql.md

```diff
@@ -95,7 +95,7 @@ Spans with the same color belong to the same service. The grey text to the right
 The Tempo data source supports streaming responses to TraceQL queries so you can see partial query results as they come in without waiting for the whole query to finish.
 
 {{% admonition type="note" %}}
-To use this experimental feature, enable the `traceQLStreaming` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
+To use this feature in Grafana, enable the `traceQLStreaming` feature toggle. This capability is enabled by default in Grafana Cloud.
```

Comment:
```suggestion
To use this feature in OSS Grafana, enable the `traceQLStreaming` feature toggle. This capability is enabled by default in Grafana Cloud.
```

Reply:
should we note that this requires v10.1 or greater? or is that already taken care of given that the docs are versioned?

Reply:
It wouldn't hurt to repeat it just to make it clear.